packages/features/handleMarkNoShow.ts

```diff
@@ -139,17 +118,143 @@ const handleMarkNoShow = async ({ bookingUid, attendees, noShowHost }: TNoShowIn
         },
       });
 
-      responsePayload["noShowHost"] = true;
-      responsePayload["message"] = "No-show status updated";
+      responsePayload.setNoShowHost(true);
+      responsePayload.setMessage("No-show status updated");
     }
 
-    return responsePayload;
+    return responsePayload.getPayload();
   } catch (error) {
     if (error instanceof Error) {
       logger.error(error.message);
     }
-    return { message: "Failed to update no-show status" };
+    throw new HttpError({ statusCode: 500, message: "Failed to update no-show status" });
   }
 };
 
+const getAttendees = async (
```

Comment:
This is actually updating the attendees in DB.
Also, best would be to have an object here with the args as properties.
```suggestion
const updateAttendees = async (
```